docs/install.rst

```diff
@@ -7,17 +7,34 @@ Installation
 Installing Ensemble Toolkit
 ===========================
 
-Installing Ensemble Toolkit using Pypi, or Git
+Installing Ensemble Toolkit using virtualenv
 ----------------------------------------------
 
 To install the Ensemble Toolkit, we need to create a virtual environment. 
 Open a terminal and run:
 
 .. code-block:: bash
 
-        virtualenv $HOME/myenv
-        source $HOME/myenv/bin/activate
+        virtualenv $HOME/ve-entk -p python3.7
```

Comment:
While working on Frontera, I saw that the suggested way to create a virtualenv with >= python3.3 is: `python3 -m venv $HOME/ve-entk` (https://docs.python.org/3/library/venv.html) should we move to this?

Reply:
These two are different libraries, venv is a standard lib and virtualenv is an external lib from pypi although I don't see much difference in the way of building a separate environment (directory e.g. ve-entk for site-packages). I am not against replacing virtualenv to -m venv. Let me know what you think.

Reply:
Maybe, in the few tests I did it worked.  But note that `venv` does behave different than virtualenv, and misses a couple of features, so maybe we use it a while and test before we recommend / document it as the default way?  If it seems stable, we can also consider using it in the agent bootstrapper.

Reply:
Fair, they are indeed different so I agree with the suggestion to test it before for a while. On Frontera and other XSEDE resource(s) (do not remember now which one, might have been stampede2/comet), with virtualenv, I was not able to update pip while I was able with venv. That was a roadblock where pip was outdated enough to fail installing RCT. Eliminating virtualenv has also the benefit to reduce of one package our dependences (a package that is not well maintaned sometimes on the HPC machines, requiring user-level installation, something that happen to me).

Reply:
Ack on all points...